### ISSUE TITLE
Unneeded process splitting when root

### ISSUE THREAD

**Issue by audreylace** [radvd-project/radvd#218]
When running as root it is seems wasteful to fork the process into 2 as both processes can still write into and modify the proc filesystem. 

Start of code that seems unneeded if the process is going to run as root anyways and not switch to another user:

https://github.com/radvd-project/radvd/blob/cf213516101c6871dd697612916ed5f4a282b7c1/radvd.c#L385

**Comment by stappersg** [comment]
On Mon, Dec 04, 2023 at 12:36:54PM -0800, Audrey Lace wrote:
> When running as root it is seems wasteful to fork the process into 2
> as both processes can still write into and modify the proc filesystem.


What is the purpose of this issue?

Please express any goal


Groeten
Geert Stappers

P.S.
Do also update  #219 & #220
-- 
Silence is hard to parse

**Comment by audreylace** [comment]
@stappersg for this issue pointing out a potential performance optimization/extra system overhead. Minimal impact at most. 

Let me know if there is a format to follow :)

**Comment by stappersg** [comment]
> Back to "What is the purpose of this issue?" If the answer is still
> 
>    _pointing out a potential performance optimization/extra system overhead_
> 
> then please close this issue

a much better response would be   _"Now that I know that my input gets feedback, will I continue on preparing a merge request"_

**Comment by stappersg** [comment]
On Sat, Aug 10, 2024 at 11:09:54PM -0700, Audrey Lace wrote:
> Closed #218 as not planned.

Thanks
